src/main/scala/com/fulcrumgenomics/fastq/DemuxFastqs.scala

```diff
@@ -44,24 +46,28 @@ import htsjdk.samtools.{SAMRecord, _}
 import scala.concurrent.forkjoin.ForkJoinPool
 
 object DemuxFastqs {
+
+  /** The name of the metrics file if none is given by the user. */
+  val DefaultDemuxMetricsFileName = "demux_barcode_metrics.txt"
+
   /** The name of the sample for unmatched reads. */
   val UnmatchedSampleId: String = "unmatched"
 
   /** The maximum # of records in RAM per writer. */
-  private val MaxRecordsInRamPerSamFileWriter: Int = 1e6.toInt
+  private[fastq] val MaxRecordsInRamPerSamFileWriter: Int = 1e6.toInt
 
   /** Decides whether or not to use asynchronous IO for the writers. This has a big performance benefit at the cost of
     * some RAM. RAM may balloon if there is a need to sort the output. */
-  private val UseAsyncIo: Boolean  = true
+  private[fastq] val UseAsyncIo: Boolean  = true
 
   /** The number of records to batch when demultiplexing in parallel. */
   private val DemuxBatchRecordsSize = 1e5.toInt
 
   /** Creates the sample output BAM path for the given sample. */
-  private[fastq] def sampleOutputBam(output: DirPath, sample: Sample): PathToBam = {
+  private[fastq] def outputPrefixFrom(output: DirPath, sample: Sample): PathToBam = {
```

Comment:
I know they're only typedefs, but could the return type get changed to `PathPrefix` as well please?